dhc/caas: increase /var/log to 2GB

seanschneeweiss opened the pull request:
The partition for /var/log is mostly used to 98%. As there is enough space available, the partition is increased to 2GB.
The build directory is adapted to the upstream used k8s v1.17.11.

sbueringer commented on .github/workflows/build.sh:
```diff
@@ -0,0 +1,29 @@
+
+DATE=$(date +%Y%m%d-%H%M)
+
+UBUNTU_VERSION="1910"
```
Do we want to drop this scripts? (I only need them on my fork)

If we want to keep them, should we keep them up-to-date?

If yes, change ubuntu and kube version

sbueringer commented on CAAS.md:
```diff
@@ -0,0 +1,89 @@
+
+# Prereqs
```
@bavarianbidi Just out of curiosity. Is the md still (mostly) up-to-date? Did you build like this locally?

sbueringer commented on images/capi/ansible/roles/containerd/templates/etc/systemd/system/containerd.service.d/boot-order.conf:
```diff
@@ -17,4 +17,9 @@
 # cloud-init boot stage managed by the cloud-final.service:
 # https://cloudinit.readthedocs.io/en/latest/topics/boot.html#final
 After=cloud-init.service
```
Do we know why we have to change this? Is it because we're doing something (like copying the containerd config) during the cloud-config unit?

If we know, should we document it here?

sbueringer commented on images/capi/ansible/roles/dhc/tasks/main.yaml:
```diff
@@ -0,0 +1,9 @@
+  when: ansible_os_family == "Debian"
+
+- name: download containerd
+  get_url:
```
do we still need / use this? I think it might be from our docker/containerd feature flag.

Or is it because we install a newer version?

sbueringer commented on images/capi/packer/os/os-ubuntu-1804.json:
```diff
@@ -0,0 +1,14 @@
+{
+  "build_name": "ubuntu-1804",
```
Could it be that the whole os subtree can be deleted?

I knew we implemented the build for Openstack in the os subfolder before upstream added the qemu build.
Afaik we're only using the qemu folders

sbueringer commented on images/capi/packer/ova/linux/ubuntu/http/base/preseed.cfg:
```diff
@@ -52,12 +52,24 @@ d-i partman-auto-lvm/guided_size string max
 # really dislikes the idea of anyone else managing memory.
 d-i partman-auto/expert_recipe string                         \
       slash ::                                                \
```
afaik ova is not used by us

sbueringer commented on images/capi/packer/qemu/qemu-ubuntu-1910.json:
```diff
@@ -0,0 +1,14 @@
+{
+  "build_name": "ubuntu-1910",
+  "distro_name": "ubuntu",
```
WDYT should we drop this? Or keep it so we can fallback if necessary? (Not sure if we have to rebuild the image in case we fall back to 1910)

sbueringer replied:
If I see correctly the makefile doesn't support 1910 atm (anymore compared to our old branch)

seanschneeweiss committed:
dhc/caas: increase /var/log partition to 2GB

The partition for /var/log is mostly used to 98%. As there is enough
space available, the partition is increased to 2GB.
The build directory is adapted to the upstream used k8s v1.17.11.
